docs/e2e-test.md

```diff
@@ -101,7 +101,7 @@ in the ephemeral cluster either before pivoting or after re-pivoting.
 
 | tests         | CAPM3 from  | CAPM3 to  | CAPI from  | CAPI to         |
 | --------------| ----------- | --------- | ---------- |---------------- |
-| v1.7=>current | v1.7.0      | main      | v1.7.3     | latest release  |
+| v1.7=>current | v1.7.0      | main      | v1.8.0-rc.1    | latest release  |
```

Comment:
CAPI from 1.8.0-rc.1? This doesn't sound right.

Reply:
ah true, lets not touch these things until GA is there
/hold

Reply:
Nice catch. Fixed now.